site/content/docs/tutorials/supervisor-without-concierge-demo.md

```diff
@@ -0,0 +1,265 @@
+
+```bash
+pinniped get kubeconfig \
+  --no-concierge \
+  --oidc-issuer "https://my-supervisor.example.com/my-issuer" \
+  --oidc-ca-bundle "supervisor_root_ca_cert.pem" \
+  --oidc-request-audience "my-cluster-342klb7h" \
+  --kubeconfig "my-admin-kubeconfig-for-this-cluster.yaml" \
+  > pinniped-kubeconfig.yaml
+```
+
+- Use `--no-concierge` to indicate that you are not using the Pinniped Concierge on this cluster.
+- The `--oidc-issuer` value should exactly match the issuer URL configured in the `kube-apiserver`'s `--oidc-issuer-url` flag and the `spec.issuer` of your Supervisor's FederationDomain.
+- The `--oidc-ca-bundle` flag is only required when the machine on which you are running this command is not going to trust your Supervisor's FederationDomain's TLS certificates, e.g. if you used a self-signed CA. This file would have the same content as the file that you provided to `kube-apiserver`'s `--oidc-ca-file` flag.
+- The `--oidc-request-audience` value should exactly match the value that you chose for the `kube-apiserver`'s `--oidc-client-id` flag.
+- The `--kubeconfig` value is the admin kubeconfig of the cluster for which you would like to generate a Pinniped-compatible kubeconfig. This is not needed when your current context is already set to the cluster.
+- The command will output the new Pinniped-compatible kubeconfig to stdout. Optionally redirect this to a file.
+
+## Example of configuring these kube-apiserver flags on kind
+
+[kind](https://kind.sigs.k8s.io) is a tool for creating and managing Kubernetes clusters on your local machine
+which uses Docker containers as the cluster's nodes. This is a convenient way to try out this feature on a local
+non-production cluster.
+
+The following steps deploy the latest release of Pinniped on kind using the local-user-authenticator component
+as the authenticator.
+
+1. Install the tools required for the following steps.
+
+   - [Install kind](https://kind.sigs.k8s.io/docs/user/quick-start/), if not already installed. For example, `brew install kind` on macOS.
+
+   - kind depends on Docker. If not already installed, [install Docker](https://docs.docker.com/get-docker/), for example `brew cask install docker` on macOS.
+
+   - This demo requires `kubectl`, which comes with Docker, or can be [installed separately](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+   - [Install the Pinniped command-line tool]({{< ref "../howto/install-cli" >}}).
+
+1. Create a kind configuration yaml file to ask kind to configure the `kube-apiserver` flags. Note that some of these
+   values will need to be adjusted as described in the comments below before using this file in the next step.
+
+   ```yaml
+   kind: Cluster
+   apiVersion: kind.x-k8s.io/v1alpha4
+   nodes:
+   - role: control-plane
+     extraMounts:
+         # Adjust this path to your CA PEM file. Use an absolute path.
+         - hostPath: /Users/ryan/supervisor_root_ca_crt.pem
```

Comment:
`supervisor_root_ca_cert.pem` would match the `pinniped get kubeconfig` example below, these do not need to be the same files but need to be the same contents

Reply:
Fixed.